client/js/models/archive.js

```diff
@@ -12,5 +17,15 @@ module.exports = {
         file: null
       }
     }
+  },
+  effects: {
+    new: function (data, state, send, done) {
+      window.alert('this should create a new archive')
+      setTimeout(() => done(), 1000)
+    },
+    load: function (data, state, send, done) {
+      window.alert('this should load the archive', data)
+      setTimeout(() => done(), 1000)
+    }
```

Comment:
i don't think we'll need a generic `load` function here. Based on the discussion in #103, we are going to server-render the archive metadata into the page, and then use webrtc peers to populate each file's content.